### DISCUSSION TITLE
Unabel to sync data sources over https since upgrade to v3.7.0

### DISCUSSION THREAD

**Discussion by kawalex** [netbox-community/netbox#14866] [Help Wanted!]
Hi,
Since upgrade to v3.7.0, I'm unable to sync my datasource on a gitlab repo using https.
The jobs are failling with error SyncError("Fetching remote data failed (TypeError):  dulwich.client.SSHGitClient() got multiple values for keyword argument  'username'")
The error is quite strange as i'm using https and not SSH.
Anyone having the same issue ? Any clue on how to fix this ?
Thanks !

**Comment by bitcollector1** [comment]
I'm on 3.7.1 and was able to sync just fine to our git repo. I have run into issues in the past where I thought it was broken and it ended up being a firewall issue.

**Reply by kawalex** [reply]
@bitcollector1, are you using HTTPS or SSH to access your repo ?

**Reply by bitcollector1** [reply]
I'm also using HTTPS for my git sync.

**Comment by jeremystretch** [comment]
Please share the settings you're using (sanitizing values where applicable), as well as the version of `dulwich` you have installed (run `pip freeze | grep dulwich` inside the Python virtual environment).

**Reply by kawalex** [reply]
Hi,
I'm using dulwich==0.21.7. The settings for the data source are ;
        {
            "id": 2,
            "url": "http://xxxxxxx:8000/api/core/data-sources/2/",
            "display": "Repo for Netbox on OCN GIT",
            "name": "Repo for Netbox on OCN GIT",
            "type": {
                "value": "git",
                "label": "Git"
            },
            "source_url": "https://xxxxxx/achelminski/repo-netbox.git",
            "enabled": true,
            "status": {
                "value": "failed",
                "label": "Failed"
            },
            "description": "",
            "comments": "",
            "parameters": {
                "branch": "main",
                "password": "xxxxxxxx",
                "username": "Netbox"
            },
            "ignore_rules": "",
            "created": "2023-06-07T12:00:03.633815Z",
            "last_updated": "2024-01-11T14:00:28.810849Z",
            "file_count": 5
        },
